requirements.txt

```diff
@@ -0,0 +1,27 @@
+absl-py==1.4.0
```

Comment:
We don't use requirements.txt, but [`pyproject.toml`](https://github.com/mlcommons/croissant/blob/main/python/mlcroissant/pyproject.toml) to manage dependencies.